### ISSUE TITLE
Mouse velocity is not updated in reactive mode when mouse stops

### ISSUE THREAD

**Issue by tangmi** [emilk/egui#108]
When egui is set in the "reactive mode", the mouse velocity will remain non-zero even after the mouse stops moving. In the "continuous mode" the mouse velocity with become zero shortly after the mouse stops moving.

**Comment by emilk** [comment]
In "reactive" mode, Egui only paints when there is something new happening. When there isn't anything happening, the last drawn frame remains. If the last thing that happened was mouse movement, then the window will show that until another frame is painted. `egui_web` manually repaints every second, so if you try the web demo ("Inspection" window at https://emilk.github.io/egui/#demo) you will see that it drops to zero the next frame.

Basically: how could egui show that the mouse isn't moving, when egui isn't being painted? It can't show the correct time of day either when it isn't being painted, but whenever such things are important to show correctly you should call `ctx.request_repaint()`.
